# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/Shape.h
# Commit: 5595064ffb60
# Full Hash: 5595064ffb6009cf14ce9b299d10856154e984e2
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2021-03-06 09:33:16
# Regressor Bug: 1689413
# File Overlap Count: 1
# Description:
#   Bug 1689413 part 10 - Stop passing proto in a few places now that we can get it from the shape. r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106980
# ==============================================================================

diff -r e15f1a6f5a38 -r 5595064ffb60 js/src/vm/Shape.h
--- a/js/src/vm/Shape.h	Fri Mar 05 19:10:08 2021 +0000
+++ b/js/src/vm/Shape.h	Fri Mar 05 19:10:08 2021 +0000
@@ -1406,8 +1406,7 @@
    * getInitialShape calls, until the new shape becomes unreachable in a GC
    * and the table entry is purged.
    */
-  static void insertInitialShape(JSContext* cx, HandleShape shape,
-                                 HandleObject proto);
+  static void insertInitialShape(JSContext* cx, HandleShape shape);
 
   /*
    * Some object subclasses are allocated with a built-in set of properties.
